### ISSUE TITLE
Kommunikation Studentenwerk bzgl Mensa-RSS-to-JSON-API

### ISSUE THREAD

**Issue by BenchR267** [HTWDD/Webservices#2]

**Comment by FestPlatin** [comment]
Kommunikation über Ticket#2017021973000016

Da die letzte Kommunikation mit dem Studentenwerk auch schon wieder über einen Monat her ist, habe ich heute nochmal nachgefragt. Ich bleib am Ball :)

**Comment by BenchR267** [comment]
Wir überlegen momentan, ob wir nicht einfach auf dem rubu einen Service aufsetzen, der auf https://github.com/kiliankoe/swdd/blob/master/speiseplan/speiseplan.go aufbaut. Dann können wir bei Änderungen schnell reagieren und haben alle Infos, die wir so brauchen.

**Comment by FestPlatin** [comment]
Solange es keine offizielle Schnittstelle gibt, wäre das eine gute Lösung und wir könnten die Apps schon mal entlasten.
Die denke die offizielle würde sich eh noch ein wenig hinziehen.

**Comment by kiliankoe** [comment]
Klar, hier (statisch und outdated, aber als Beispiel). https://users.ifsr.de/~koeltzsch/speiseplan.json

**Comment by BenchR267** [comment]
Wir hatten noch die Überlegung, die Speisen noch nach Mensen zu gruppieren

**Comment by kiliankoe** [comment]
Klingt sinnvoll, aktuell ist das quasi nur ne Übersetzung von invalidem RSS zu JSON.

**Comment by FestPlatin** [comment]
Das wäre auf jeden Fall noch gut, bzw gleich als Parameter das nur die entsprechenden zurückgeliefert werden?
Für Android wären eigentlich auch noch die Wochenpläne interessant. Hattet ihre dahingehend schon mal was gemacht?
